.github/workflows/container-images-cd.yml

```diff
@@ -30,12 +30,12 @@ jobs:
 
         steps:
             - name: Check out
-              uses: actions/checkout@v3
+              uses: actions/checkout@v4
```

Comment:
This is the latest major version of this action at the time of this pull request's creation.